### PR TITLE
Fix part #10713: High-fi of Stories Completed Portrait mode.

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR focuses on hi-fi part of #1073  Stories Completed portrait mode.

### Mock
https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/f54ae7d1-9530-44d6-8783-21559b13635c/UP-Stories-Completed-/

- [x] Passes Accessibility test

### Screenshot
Screenshot_20200515-160256

### CODE REVIEW

File: app/src/main/res/layout/completed_story_item.xml [hunk 1]
```diff
@@ -1,81 +1,173 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--<layout xmlns:android="http://schemas.android.com/apk/res/android"-->
+<!--  xmlns:app="http://schemas.android.com/apk/res-auto">-->
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/layout/completed_story_item.xml]: Please remove this if not required.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

File: app/src/main/res/layout/completed_story_item.xml [hunk 2]
```diff
@@ -1,81 +1,173 @@
...
-          android:textSize="14sp" />
-      </LinearLayout>
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="4dp"
+        android:layout_marginEnd="8dp"
+        android:fontFamily="sans-serif-light"
+        android:paddingBottom="12dp"
+        android:textAllCaps="true"
+        android:text="@{viewModel.completedStory.topicName}"
+        android:textColor="@color/oppiaStrokeBlack"
```
Comment (rt4914) [inline app/src/main/res/layout/completed_story_item.xml:85]: Introduce max lines and min lines for this text view.

**Review by rt4914 — CHANGES_REQUESTED** [review]
The UI looks like this please fix this:
Screenshot_1589778756

File: app/src/main/res/layout/completed_story_item.xml (hunk 1 above)
Comment (veena14cs) [reply app/src/main/res/layout/completed_story_item.xml]: done.

File: app/src/main/res/layout/completed_story_item.xml (hunk 2 above)
Comment (veena14cs) [reply app/src/main/res/layout/completed_story_item.xml:85]: Added max and min lines.

**Comment by veena14cs** [comment]
> The UI looks like this please fix this:
> Screenshot_1589778756

Fixed this issue.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**